src/handlers/users/UserHandler.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+const HandlerBase = require('../HandlerBase');
+const UserMapper = require('../../maps/UserMapper');
+
+module.exports = class UserHandler extends HandlerBase {
+  constructor() {
+    super();
+  }
+
+  async getUsers(req, res) {
```

Comment:
Isn't everything already async with node, how is the 'async' helping. just asking

Reply:
It's because of this line
` const results = await UserMap.find({});`
it resolves a promise to the result without a `then()`